### PR TITLE
Added LotusScript

### PR DESCRIPTION
**Pull request by icositetrachoron-programmer**

### CODE REVIEW

File: l/lotusscript.lss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ Sub Click(Source As Button)
```
Comment (Richienb) [inline l/lotusscript.lss:1]: Does the statement need to be enclosed in this function or can the Print statement be usable on its own?

Comment (icositetrachoron-programmer) [reply l/lotusscript.lss:1]: It has to be enclosed in the function. Otherwise I don't think there's another way to trigger the script.

Comment (Richienb) [reply l/lotusscript.lss:1]: In that case, we should run it on initialisation.

Comment (Richienb) [inline l/lotusscript.lss:1]:
```suggestion
 Sub Initialize
```

**Review by Richienb — CHANGES_REQUESTED** [review]

**Pull request closed without merging**